tensornetwork/linalg/tests/initialization_test.py

```diff
@@ -100,6 +100,33 @@ def test_ones(backend):
     npI = backend_obj.ones(shape, dtype=dtype)
     np.testing.assert_allclose(tnI.array, npI)
 
+def test_ones_like(backend):
+    """Tests tensornetwork.ones_like against np.ones_like"""
+    backend_obj = backends.backend_factory.get_backend(backend)
+    shape = (2, 4, 1)
+    n = np.eye(2)
```

Comment:
test with the relevant array type for each backend. also vary the dtype

Reply:
As I saw the tests run on each backend type and at line 122 the for loops over all dtypes and each run a new objTensor is created, is that you meant?